### ISSUE TITLE
Support for PHP 8

### ISSUE THREAD

**Issue by panakour** [OFFLINE-GmbH/oc-mall-plugin#743]
Is there any plan to make the mall working using php 8?

**Comment by tobias-kuendig** [comment]
I have not tested it yet, what does not work specifically?

**Comment by panakour** [comment]
I've found some third party dependencies which needs to be updated to support php 8.
```
hashids/hashids
league/omnipay
omnipay/paypal
omnipay/stripe
barryvdh/laravel-dompdf
```

**Comment by tobias-kuendig** [comment]
PHP 8 support is now available in version 1.14.0: https://offline-gmbh.github.io/oc-mall-plugin/changelog/1.14.0.html

Thank you for your feedback @panakour!
